components/automate-deployment/pkg/server/api_token.go

```diff
@@ -33,6 +35,7 @@ func generateAdminToken(ctx context.Context,
 	req *api.GenerateAdminTokenRequest, connFactory *secureconn.Factory,
 	authNAddress, authZAddress string) (*api.GenerateAdminTokenResponse, error) {
 
+	ctx = auth_context.NewOutgoingContext(auth_context.NewContext(ctx, []string{"tls:service:deployment-service:internal"}, []string{}, "", "", "v2.1"))
```

Comment:
is this better than adding this call as an exempted function to the interceptor? (also, tokens! :shakes_fist:)

Reply:
nope, i'm all the wrong. carry on.